bouncer.go

```diff
@@ -362,27 +562,52 @@ func getToken(bouncer *Bouncer) error {
 	return fmt.Errorf("getToken statusCode:%d", login.Code)
 }
 
+func refreshLapiToken(bouncer *Bouncer) {
```

Comment:
Customize the getToken function to return a string that can be the CAPI_CrowdsecKey or LAPI_JWT, and in params the url
func get(bouncer, url) (string, error) {...}
That will save us from copy paste function refreshLapiToken

Reply:
Yep, good point. Done.